imagersite/imagersite/settings.py

```diff
@@ -101,7 +113,25 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/1.8/howto/static-files/
 
-STATIC_ROOT = os.path.join(BASE_DIR, 'static')
+# STATIC_ROOT = os.path.join(BASE_DIR, 'static')
 STATIC_URL = '/static/'
+STATICFILES_DIRS = [
+    os.path.join(BASE_DIR, 'static'),
+]
```

Comment:
You probably actually want this to be `os.path.join(BASE_DIR, 'imagersite/static')` so that you are using a static directory in the site folder, not up at the project level.